src/translators/AccountTranslator.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import type { Account, AccountMutator } from '../index';
+import DefaultTranslator from './DefaultTranslator';
+
+const formatPhoneNumber = (number: ?number): ?string => {
```

Comment:
I would much rather have this done at the UI layer so it makes sense to the user

https://github.com/sanniassin/react-input-mask/issues/9

It's also much safer than what you're doing. There is a ton of talk on stackoverflow on how to properly parse phone numbers with a regex.

I think in the app source code I actually strip out all of the non numeric characters before I send to the server.